src/Glpi/Asset/Capacity/HasVolumesCapacity.php

```diff
@@ -51,6 +52,12 @@ public function getIcon(): string
         return Item_Disk::getIcon();
     }
 
+    #[Override]
+    public function getDescription(): string
+    {
+        return __("List storage volumes found by automatic inventory");
```

Comment:
Not only from inventory
```suggestion
        return __("List storage volumes");
```